chime-notifier/README.md

```diff
@@ -0,0 +1,51 @@
+## AWS Health Chime Notifier
+
+### Description
+
+This tool can be used to post alerts to a Chime room when AWS Health events are generated by using AWS Lambda and Amazon CloudWatch Events. 
+
+### Chime Setup
+Follow these steps to configure the webhook in Chime:
+
+1. Click the cog wheel from your Chime room 
+
+2. Select "Manage WebHooks".
+
+3. Create Webhook
+
+4. Copy the webhook URL and use it in the AWS Setup section that follows.
+
+
```

Comment:
A reminder to treat the webhook like a password (i.e. don't save it somewhere public) would be good

Reply:
updated the readme